src/frontend/components/injector-tree/injector-tree.ts

```diff
@@ -63,7 +63,7 @@ export class InjectorTree implements OnChanges {
   private unFocusNode = () => {
     this.focusedComponent = -1;
     this.focusedDependency = -1;
-  };
+  }
```

Comment:
Semicolon at end of the assignment is more correct. In general I prefer we not leave out semicolons.